framework/src/st_things/things_stack/framework/things_req_handler.c

```diff
@@ -271,7 +264,7 @@ static OCEntityHandlerResult get_provisioning_info(things_resource_s *target_res
 {
 	OCEntityHandlerResult eh_result = OC_EH_ERROR;
 
-	const char *device_rt = dm_get_things_device_type(0);
+	const char *device_rt = dm_get_things_device_type("0");
```

Comment:
i don't understand meaning of "0"
i think in this logic. you don't need any parameters
because this st_things has only one device and have global variable.

Reply:
yes. It is correct. I will make the changes.